ixmp/testing/__init__.py

```diff
@@ -579,6 +640,7 @@ def _platform_fixture(
     tmp_env: os._Environ[str],
     test_data_path: Path,
     backend: Literal["jdbc", "ixmp4"],
+    worker_id: str,
```

Comment:
Since this is universally available with xdist, I don't think it needs to be passed in as an argument.